sdk/storage/inc/blobs/protocol/blob_rest_client.hpp

```diff
@@ -354,6 +363,12 @@ namespace Azure { namespace Storage { namespace Blobs {
     std::string Name;
   }; // struct BlobPrefix
 
+  struct BlobRetentionPolicy
+  {
+    bool Enabled = false;
+    Azure::Core::Nullable<int32_t> Days;
```

Comment:
Retention policy itself can be nullable but not `Days` right?

Reply:
The swagger is having different behavior with the doc, with swagger stating the parameter being required but doc does not:

Required only if a retention policy is enabled. Indicates the number of days that metrics or logging data should be retained. All data older than this value will be deleted. The minimum value you can specify is 1; the largest value is 365 (one year).
--

Reply:
I think we should follow the doc, not swagger.

Reply:
oh great, another testimony that swagger is not perfect. i'm going to write it down.